include/zephyr/drivers/usbc/usbc_vbus.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2022 The Chromium OS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USBC VBUS device APIs
+ *
+ * This file contains the USBC VBUS device APIs.
+ * All USBC VBUS measurment and control device drivers should
+ * implement the APIs described in this file.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_USBC_VBUS_H_
+#define ZEPHYR_INCLUDE_DRIVERS_USBC_VBUS_H_
+
+/**
+ * @brief USBC VBUS API
+ * @defgroup usbc_vbus_api USBC VBUS API
+ * @ingroup io_interfaces
+ * @{
+ */
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/usbc/usbc_tc.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+__subsystem struct vbus_driver_api {
```

Comment:
I think `__subsystem` is only needed for userspace, doesn't seem this API supports it.

Reply:
Done